DataFormats/FEDRawData/doc/FEDRawData.doc

```diff
@@ -15,6 +15,72 @@
 Classes to store raw data blocks in the event.
 FED data blocks are required to have correct padding (64 bit).
 
+<!-- This table must be created with DataFormats/FEDRawData/doc/createFEDtable.py -->
```

Comment:
Ok... Let me then revert back to the previous situation. Sorry, @mommsen, that won't work.